files/usr/share/cinnamon/cinnamon-settings/modules/cs_info.py

```diff
@@ -93,6 +93,16 @@ def getProcInfos():
     return result
 
 
+def getSystemIcon():
+    schema = Gio.Settings(schema="org.cinnamon")
+
+    if schema.get_boolean("system-icon-enabled"):
+        iconPath = schema.get_string("system-icon-path")
+        return iconPath
+    else:
+        return
+
+
```

Comment:
Let's just have the single 'system-icon-path' key - we don't need a separate 'enabled' setting.  If the path is empty, it's disabled.

Also, if your function returns something, it should always return something - even though in this case it amounts to the same thing, explicit is better, and a good practice. In this instance, it's best to return `None`.